src/App.tsx

```diff
@@ -1,14 +1,42 @@
 /* eslint-disable max-len */
-import React from 'react';
+import React, { useEffect, useState } from 'react';
 import 'bulma/css/bulma.css';
 import '@fortawesome/fontawesome-free/css/all.css';
 
 import { TodoList } from './components/TodoList';
 import { TodoFilter } from './components/TodoFilter';
+import { getTodos } from './api';
 import { TodoModal } from './components/TodoModal';
 import { Loader } from './components/Loader';
+import { getFilteredTodos } from './taskFilter';
+import { Todo } from './types/Todo';
 
 export const App: React.FC = () => {
+  const [tasks, setTasks] = useState<Todo[] | null>(null);
```

Comment:
As a matter of preference, I'd rather have an empty array here for the initial state.